src/niftyone/figures/generator.py

```diff
@@ -114,12 +135,35 @@ def generate(self, record: pd.Series, out_dir: Path, overwrite: bool) -> None:
 
         img_path = Path(record["finfo"]["file_path"])
         logging.info("Processing: %s", img_path)
-        img = nib.nifti1.load(img_path)
-        img = noimg.to_iso_ras(img)
+
+        signature = inspect.signature(self.view_fn)
```

Comment:
The use of inspection here feels a bit awkward.

Reply:
Agreed (and see note below). Outside of needing to pass the Path in for one dwi figure (will probably rewrite and then strip this out), this isn't really needed anymore.

Reply:
Taken out in 85e1596.